### PR TITLE
Add more connections between algorithms

### PR DESCRIPTION
**Pull request by DelazJ**
Goal:

Ticket(s): #

- [ ] Backport to LTR documentation is required

### CODE REVIEW

File: source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst [hunk 1]
```diff
@@ -2973,6 +3005,8 @@ and longitude values reversed.
 
 |checkbox| Allows :ref:`features in-place modification <processing_inplace_edit>`
 
+.. seealso:: :ref:`qgisreverselinedirection`
```
Comment (havatv) [inline source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst]: These are not really related, are they?

Comment (DelazJ) [reply source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst]: Yeah, I was in doubt here. But I think I'll remove this one. How about the other way?

Comment (DelazJ) [reply source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst]: Actually I think I'll scratch this connection and link swapxy to rotate and translate. Opinions...

Comment (havatv) [reply source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst]: Sounds sensible to link it to other coordinate transformations, yes.
